core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -158,9 +157,10 @@
                                    gracefulShutdownQuietPeriod, "gracefulShutdownQuietPeriod");
 
         requireNonNull(blockingTaskExecutor, "blockingTaskExecutor");
-        if (!(blockingTaskExecutor instanceof TimedExecutorService)) {
-            blockingTaskExecutor = ExecutorServiceMetrics.monitor(meterRegistry, blockingTaskExecutor,
-                                                                  "armeriaBlockingTaskExecutor");
+        if (!ExecutorServiceMetrics.isMonitoredExecutor(blockingTaskExecutor)) {
+            blockingTaskExecutor =
+                    ExecutorServiceMetrics.monitor(meterRegistry, blockingTaskExecutor,
+                                                   "armeriaBlockingTaskExecutor", "armeria.executor");
```

Comment:
Now that we have our own meter name, could we remove `armeria` from the executor name?

Reply:
Yes! 😎